content/desktop/synchronized-file-sharing.md

```diff
@@ -1,73 +1,55 @@
 ---
-title: Synchronized file sharing
-description: Get started with Synchronized file sharing on Docker Desktop.
-sitemap: false
+title: Synchronized File Shares
+description: Get started with Synchronized File Shares on Docker Desktop.
+keyword: mutagen, file sharing, docker desktop, bind mounts
 ---
 
-> **Early Access**
+> **Note**
 >
-> Synchronized file sharing is an early-access feature. 
->
-> If you would like to be considered for Synchronized file sharing testing, you can
-> [sign up for the early access program](https://www.docker.com/docker-desktop-preview-program/).
-{ .restricted }
+> Synchronized File Shares is available for customers with a Docker subscription. 
```

Comment:
I know this was already addressed below, but the plurality mismatch between the singular feature and its plural name does make for a tough read.

I noticed that the Docker Extensions documentation takes an approach where they primarily refer to "extensions" (not the "Docker Extensions" feature name) to avoid a mismatch.  Maybe that's an approach we could take on a future revision (referring to "shares" or "file shares").

Though even the D.E. docs aren't always consistent (e.g. "Docker Extensions that are not in the Marketplace haven't gone through Docker's review process." and "Docker Extensions is switched on by default.").